DisplayCAL/xh_filebrowsebutton.py

```diff
@@ -43,6 +43,10 @@ def DoCreateResource(self):
             name=self.GetName(),
         )
         self.SetupWindow(w)
+        if self.GetBool("hidden") and w.Shown:
+            from log import safe_print
```

Comment:
This is unnecessary, try to group your imports. Conditional imports may make sense to address different architectures or if something like a fallback can be used, but not here.
https://peps.python.org/pep-0008/#imports

Reply:
it's what I thought also (re wxpython), couldn't find anything in https://wxpython.org/pages/changes/index.html though and there seems relatively little info on wxpython around so workaround seemed fair to me

Reply:
it's fair point, running those locally I see the segfault is in wx

```
[50899.035449] python[431559]: segfault at 140 ip 00007fd49b8a163d sp 00007ffdd5ccdbf0 error 4 in libwx_baseu-3.2.so.0[7fd49b87c000+187000] likely on CPU 8 (core 2, socket 0)
```

running the same on develop doesn't fail

I haven't seen any such failure running the actual app with my changes

on my system (debian trixie), `libwx_baseu-3.2.so.0` appears to be part of

```
libwxbase3.2-1/testing,unstable,now 3.2.2+dfsg-2 amd64 [installed,automatic]
  wxBase library (runtime) - non-GUI support classes of wxWidgets toolkit
```

in my venv I have

`wxPython 4.2.0 gtk3 (phoenix) wxWidgets 3.2.0`

Reply:
I pushed a change to move the import and now it works :shrug: